docs/proposals/extender-readiness-check.md

```diff
@@ -0,0 +1,78 @@
+# Proposal: Scheduler-extender readiness check
+
+Last updated: 08.07.2021
+
+
+## Abstract
+CSI has no way to verify scheduler-extender deployment status.
+The goal - create an opportunity for user to understand: is scheduler-extender installed completely and able to work or not.
+
+## Background
+Currently, there are 2 patching methods depends on platforms.
```

Comment:
`Currently CSI Operator doesn't wait for scheduler extender to be completely installed on the system`

Reply:
fixed